fibbingnode/misc/mininetlib/ipnet.py

```diff
@@ -155,6 +155,17 @@ def start(self, timeout=None):
         if self.wait_for_convergence:
             self._convergence_test(timeout)
 
+        # Add black holes for UDP flows
+        log.info('*** Adding IPTABLES RULES\n')
+        for h in self.hosts:
+            # Drop all UDP traffic
+            cmd = "iptables -A INPUT -s {0} -p udp -j DROP".format(self.ipBase)
```

Comment:
This is just a black hole (ie consumer) at the Mininet hosts, useful for the creation of UDP flows in the network.

Sorry, this shouldn't be here, but in the TE code instead. :+1: